sllurp/llrp_decoder.py

```diff
@@ -54,6 +54,7 @@
 TVE_PARAM_TYPE_MAX = 127
 TYPE_CUSTOM = 1023
 VENDOR_ID_IMPINJ = 25882
+VENDOR_ID_MOTOROLA = 161
```

Comment:
Just for reference/memory, maybe you can add another vendor like:
VENDOR_ID_MOTOROLA_SIRIT = 24831
Just to keep a reference as it is also reported as "manufacturer" id by the Zebra reader

Reply:
I added VENDOR_ID_SIRIT, as it is registered to IANA. Doesn't make sense to include motorola in the name.